sentry_sdk/integrations/gcp.py

```diff
@@ -159,22 +159,22 @@ def _get_google_cloud_logs_url(initial_time):
     Returns:
         str -- Google Cloud Logs Console URL to logs.
     """
-    hour_ago = initial_time - timedelta(hours=1)
+    hour_ago = final_time - timedelta(hours=1)
 
     url = (
         "https://console.cloud.google.com/logs/viewer?project={project}&resource=cloud_function"
         "%2Ffunction_name%2F{function_name}%2Fregion%2F{region}&minLogLevel=0&expandAll=false"
-        "&timestamp={initial_time}&customFacets=&limitCustomFacetWidth=true"
-        "&dateRangeStart={timestamp_start}&dateRangeEnd={timestamp_end}"
-        "&interval=PT1H&scrollTimestamp={timestamp_current}"
+        "&timestamp={initial_time}Z&customFacets=&limitCustomFacetWidth=true"
+        "&dateRangeStart={timestamp_start}Z&dateRangeEnd={timestamp_end}Z"
+        "&interval=PT1H&scrollTimestamp={timestamp_current}Z"
     ).format(
         project=environ.get("GCP_PROJECT"),
         function_name=environ.get("FUNCTION_NAME"),
         region=environ.get("FUNCTION_REGION"),
```

Comment:
This was added in a previous commit, but since we're fixing the link, I ask -- is it guaranteed that those variables are always set in the environment?

If they are not set, they will produce `None` in the middle of the URL, possibly not a working URL to the GCP console. In that case, it seems more reasonable to omit the URL altogether.

Reply:
These environment variables are reserved for Python 3.7 runtime environment by Google Cloud platform as per the documentation.
Link : https://cloud.google.com/functions/docs/env-var

Reply:
👍